lib/aquatone/collectors/crtsh.rb

```diff
@@ -0,0 +1,21 @@
+module Aquatone
+  module Collectors
+    class Crtsh < Aquatone::Collector
+      self.meta = {
+        :name         => "crtsh",
+        :author       => "Joel (@jolle)",
+        :description  => "Uses crt.sh by COMODO CA to find hostnames"
+      }
+
+      def run
+        response = get_request("https://crt.sh/?dNSName=%25.#{url_escape(domain.name)}")
+
+        parsed = parse_html(response.body)
+
+        parsed.css('table').css('table').css('tr').css('td:nth-child(4)').map do |column|
```

Comment:
Do you think you could do the extraction with regular expressions instead? I would like to avoid having Nokogiri as a dependency as the native extension for the gem often fails to compile for people, which results in a lot of GitHub Issues and frustrated users. :)